projects/core/src/cart/facade/cart.service.ts

```diff
@@ -184,15 +176,19 @@ export class CartService {
 
   getEntry(productCode: string): Observable<OrderEntry> {
     return this.store.pipe(
-      select(fromSelector.getEntrySelectorFactory(productCode))
+      select(fromSelector.getCartEntrySelectorFactory(productCode))
     );
   }
 
-  isCreated(cart: Cart): boolean {
-    return cart && !!Object.keys(cart).length;
+  private isCreated(cart: Cart): boolean {
+    return cart && typeof cart.guid !== 'undefined';
   }
 
-  isEmpty(cart: Cart): boolean {
-    return cart && !cart.totalItems;
+  /**
+   * Cart is incomplete if it contains only `guid` and `code` properties, which come from local storage.
+   * To get cart content, we need to load cart from backend.
+   */
+  private isIncomplete(cart: Cart): boolean {
+    return cart && Object.keys(cart).length <= 2;
```

Comment:
also need to check the `Object.keys(cart).length > 0`. It should not be empty.

Reply:
This condition is not necessary